packages/output-components-as-svgr/src/index.ts

```diff
@@ -9,7 +9,8 @@ type Options = {
     output: string;
     getDirname?: (options: FigmaExport.ComponentOutputterParamOption) => string;
     getComponentName?: (options: FigmaExport.ComponentOutputterParamOption) => string;
-    getFileExtension?: (options: FigmaExport.ComponentOutputterParamOption) => string;
+    getFileExtension?: (options?: FigmaExport.ComponentOutputterParamOption) => string;
+    getExportTemplate?: (options?: FigmaExport.ComponentOutputterParamOption) => string;
```

Comment:
`options` should be always set.

```suggestion
    getFileExtension?: (options: FigmaExport.ComponentOutputterParamOption) => string;
    getExportTemplate?: (options: FigmaExport.ComponentOutputterParamOption) => string;
```